lib/stdlib/doc/src/gen_event.xml

```diff
@@ -245,6 +245,16 @@ gen_event:stop             ----->  Module:terminate/2
                   Which term depends on the error.</p></item>
             </list>
           </item>
+          <item>
+              <p>If the calling process exits later, the event manager will
+              delete all handlers the calling process supervises (and call
+              <seemfa marker="#Module:terminate/2">
+              <c>Module:terminate({'EXIT', Pid, Reason}, State)</c></seemfa>),
+              then call
+              <seemfa marker="#Module:handle_info/2">
+              <c>Module:handle_info({'EXIT', Pid, Reason}, State)</c></seemfa> for each
+              remaining handler.</p>
+          </item>
```

Comment:
Can you please combine this bullet with bullet above starting with "<item>If the calling process later terminates" ?
I think they're describing same scenario and there is no point in splitting them.

Reply:
You're right, I missed that.